plugins/generator-1.18.2/forge-1.18.2/templates/livingentity/livingentity.java.ftl

```diff
@@ -612,6 +612,18 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
    	}
 	</#if>
 
+	<#if data.solidBoundingBox>
+	@Override
+	public boolean canCollideWith(Entity entity) {
+		return true;

```

Comment:
You should make it as a condirion procedure directly instead of requiring another PR (and converter) in the future.

Reply:
Why would it require a converter? Are you planning on changing the entity GUI?

Reply:
I mean, if we keep it like you did, but then in the future we want to chabge it to a conditional parameter (because it gives more possibilities), it will require a converter to not break older workspaces. However, if you change it now, no converter will be needed.

Reply:
I guess that's fine, but where in the GUI should I put the condition selector?